Update "Install .NET 6.0 SDK" Tip & Misc

lace-wing opened the pull request:
## Why Making The Changes?

The .NET 6.0 SDK is no longer shipped with Visual Studio 2022, and there is no option to install it through Visual Studio Installer but by manual installation. However the 'first time launch' tip was still saying that "the easiest way is to install VS 2022", which is misleading. Many new modders were confused by the situation that they have VS 2022 while the tip kept showing up. This PR fixes the English and Chinese entries by alternating it, so that it suggests users to install it manually.

P.S. I don't know why the PR includes irrelevant commits, but file content changed is all right so...

lace-wing committed:
steam page schinses loc

lace-wing committed:
added a missing double quote

lace-wing committed:
Merge branch 'tModLoader:1.4.4' into 1.4.4

lace-wing committed:
update .NET 6.0 SDK installation tip

lace-wing committed:
sync, resolve conflitcs and improve a bit

Solxanich commented on patches/tModLoader/Terraria/Localization/Content/en-US/tModLoader.json:
```diff
@@ -415,8 +415,8 @@
 		"MSPortToLatest": "Run tModPorter",
-		"MSFirstLaunchModderWelcomeMessage": "Making mods for tModLoader in Visual Studio now requires the .NET 6 SDK to be installed. After installing, you may need to restart tModLoader. This is most easily installed by installing Visual Studio 2022.\n\nThe Migration Guide found on Github will teach the code changes necessary to port mods to 1.4. The guide also teaches how to use tModPorter to migrat ...
-		"DownloadNetSDK": "Download .NET SDK",
+		"MSFirstLaunchModderWelcomeMessage": "Making mods for tModLoader in Visual Studio now requires the .NET 6 SDK to be installed. After installing, you may need to restart tModLoader. It is no longer shipped with Visual Studio 2022, so you have to install it manually. Click the \"Download .NET SDK\" bottun below to download its installer.\n\nThe Migration Guide found on Github will teach the code  ...
```
```suggestion
		"MSFirstLaunchModderWelcomeMessage": "Making mods for tModLoader in Visual Studio now requires the .NET 6 SDK to be installed. After installing, you may need to restart tModLoader. It is no longer shipped with Visual Studio 2022, so you will have to install it manually. Click the \"Download .NET SDK\" button below to download the installer.\n\nThe Migration Guide found on Github covers the code changes necessary to port mods to 1.4.4. The guide also teaches how to use tModPorter to migrate a large portion of the mod for you.",
```

lace-wing committed:
Merge branch 'tModLoader:1.4.4' into 1.4.4

lace-wing committed:
change as requested + misc translations

JavidPack committed:
Split welcome to develop mods menu and install dotnet 6 message. Fix spanish loading.

JavidPack commented:
I've updated the message and split it into 2 different messages, it was a bit confusing and outdated. A welcome message and a dotnet 6 sdk not found message. 

If you could update the Chinese to match, I'll get this merged.

lace-wing committed:
update welcome and .net sdk notfound note as they were splited

lace-wing commented:
> I've updated the message and split it into 2 different messages, it was a bit confusing and outdated. A welcome message and a dotnet 6 sdk not found message.
> 
> If you could update the Chinese to match, I'll get this merged.

Done.
